fault_injection/fi_gear/dummy/README.md

```diff
@@ -0,0 +1,14 @@
+# Template drivers for FI gear
+
+This directory includes dummy drivers for FI gear that can be used as a
+template to create new drivers.
+All drivers need to implement these functions:
+- init:
+    - Initializes the FI gear.
+- generate_fi_parameters:
+    - Generate parameters for the FI parameter sweep. This can happen either
+      randomly or deterministically.
+- arm_trigger:
+    - Configure the FI gear with the current FI parameters and arm the trigger.
+- reset:
+    - After a power cycle of the DUT, some FI gear also need to reset it.
```

Comment:
I am not sure I understand this. Do you mean the FI gear needs to reset the DUT?

Reply:
No, the FI gear itself (i.e., Husky needs a re-initialization) needs a reset after resetting/reprogramming OpenTitan/FPGA. Other FI gear (LFI, ...) does not need to implement this.

Reply:
Ah ok, thanks for explaining. So the sentence should probably read, `...some FI gear also needs to be reset.`